content/artifacts/highlight-active-artifact-planets/plugin.js

```diff
@@ -0,0 +1,173 @@
+// Display the planet with active artifact, specially for cannon.
+
+import { 
+  BiomeNames
+} from "https://cdn.skypack.dev/@darkforest_eth/types";
+
+import {
+  coords
+} from 'https://plugins.zkga.me/utils/utils.js';
+
+import {
+  ArtifactType,
+  ArtifactTypeNames
+} from "https://cdn.skypack.dev/@darkforest_eth/types";
+
+const ARTIFACT_TYPES = Object.values(ArtifactType).filter((type) => type !== ArtifactType.Unknown).map((type) => ({ value: type, text: ArtifactTypeNames[type] }));
+
+
+function getPlayerColor(ethAddress) {
+  return df.getProcgenUtils().getPlayerColor(ethAddress);
+}
+
+
+function drawArc(ctx, planet, color, artifactTypes){
+  if(!planet) {
+    return false;
+  }
+
+  let artifact = df.getActiveArtifact(planet);
+  if (!artifact) {
+    return false;
+  }
+
+  if (artifactTypes.indexOf(artifact.artifactType ) == -1)  {
+    return false;
+  }
+
+  let percent = 1;
+  if (artifact.artifactType == 7) {
```

Comment:
i assume 7 is photoid cannon?
i did not test it, but did you try something like this:
`artifact.artifactType === ArtifactType.PhotoidCannon`
it makes it more clear and if in the future the number changes it will not need an update
below is another use of the number 7

Reply:
Oh, yes.  I just didn't find the "ArtifactType.PhotoidCannon"
ArtifactType.PhotoidCannon is equal to 7 when I debug the code.
I have fixed it, thanks.